buildAll.proj

```diff
@@ -32,6 +42,17 @@
                       src\Microsoft.ServiceFabric.Services\packages.config;
                       src\Microsoft.ServiceFabric.Services.Remoting\packages.config;
                       src\Microsoft.ServiceFabric.Services.Wcf\packages.config"/>
+ 
+    <TestPackagesConfigs Include="nuprojs\packages.config;
```

Comment:
Done. Not sure why I'm now reviewing, I can't see where your changes are (if you made any)